prompt-service/src/unstract/prompt_service/plugins/table_extractor/src/extractor/core.py

```
@@ -0,0 +1,628 @@
+                        app.logger.info(f"Invalid row: {json_row}")
+                        continue
+                valid_values = 0
+                for key in keys:
+                    if json_row[key] != "":
+                        valid_values += 1
+                if valid_values < 3 or valid_values < len(keys) / 2:
+                    # 4 since first two columns are page and line number
+                    app.logger.info(f"Invalid row: {json_row}")
+                    continue
+            except Exception:
+                continue
```

Ruff (0.15.21)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)